front/src/components/ScheduleAndShiftsCreationComponents/RenderShiftTimes.tsx

```diff
@@ -0,0 +1,13 @@
+import { format } from 'date-fns';
+import {isSameDay} from "./SameDateCheck.ts";
+import {DisplayShiftTime} from "./ShiftTypesAndTimeDisplayElements.tsx";
+
+export const RenderShiftTime = ({dateToCheck, timeToRender, startOrEndTimeLabel}: {dateToCheck: Date, timeToRender: Date, startOrEndTimeLabel: string}) => {
```

Comment:
components are no verbs. they are nouns. ie `ShiftTimePane`

also i think the param names suck.
and define a `type` for them. more than two is a good rule of thumb